src/subcommand/wallet/sign.rs

```diff
@@ -64,6 +98,8 @@ impl Sign {
 
     Ok(Some(Box::new(Output {
       address: address.as_unchecked().clone(),
+      inscription: self.inscription,
+      output: self.output,
       message: self.message.clone(),
```

Comment:
Let's remove this, since there's always a message that's being signed, but it would be unreasonable to include a very large file.